<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MySerial</class>
 <widget class="QWidget" name="MySerial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MySerial</string>
  </property>
  <widget class="QLabel" name="label_port">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>串口：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_baudrate">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>74</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>波特率：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_databits">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>130</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>数据位：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_parity">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>190</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>校验方式：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_stopbits">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>246</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>停止位：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_port">
   <property name="geometry">
    <rect>
     <x>668</x>
     <y>20</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_baudrate">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>74</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>115200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>38400</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_databits">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>130</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_parity">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>190</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Even</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Odd</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_stopbits">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>246</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_openclose">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>320</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>打 开 串 口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_send">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>400</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>发 送</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>561</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>561</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
